Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md

```diff
@@ -0,0 +1,301 @@
+The caster is enveloped in shadows making it somewhat more difficult to clearly distinguish details of the caster by sight.  
+*(Base 3)*  
+
+**Jackfrost says fuck you particularly**  
+PeIg 10  
+**R:** Voice, **D:** Mom, **T:** Ind  
+Causes the target to feel a chill and lose body heat. The target loses a Fatigue level.  
+**Mastery:** Multi  
+*(Base 4, Voice +2)*
+
+**Enough**  
+PeIg lvl 15  
+**R:** Voice, **D:** Mom, **T:** Ind  
+Extinguish a fire, cooling the ashes to merely warm.  
+*(Base 4, Voice +2, Size +1)*
+
+**The Cold Never Bothered Me Anyway**  
+PeIg 25  
+**R:** Voice, **D:** Mom, **T:** Part  
+Cools the air within ten paces of the target, leaving it slightly below freezing. All nonliving things are chilled thoroughly, not just on the surface. All living things (except you, if youcast with a Rego requisite) lose one Fatigue level. House fires become as small as campfires, bonfires become as small as torch fires, and campfires and smaller fires go out.  
+**Mastery:** Resistance  
+*(Base 4, voice +2, part +1, size +2)*
+
+## Rego Ignem
+
+**Animating the Fire Wyvern**  
+ReIg lvl 20  
+**R:** Sight, **D:** Mom, **T:** Ind  
+Gathers fire within sight range, shapes it into a wyvern. Send the wyvern to a target area/individual causing +15 damage. If aiming roll needed: +12. s.80 Aiming = Per + Fin + Dice + bonus.   
+*(Base 3, Sight +3, Size +2)*
+
+# Imaginem
+
+## Creo Imaginem
+
+## Intellego Imaginem
+
+## Muto Imaginem
+
+## Perdo Imaginem
+
+## Rego Imaginem
+
+**Heh you cant find me**  
+TeFo lvl 10 
+**R:** Touch, **D:** Mom, **T:** Ind  
+Your image appears up to 1 pace from where you actually are, so attacks aimed there are likely to miss you. Whenever the image is successfully struck, it disappears and reappears in another spot. In combat, early attacks are aimed at the image, and so automatically miss you. Once the image has been hit once, you have a +9 bonus to Defense, as attackers must try to find you in a fairly large area.  
+*(Base 2, +2 Sun, +1 changing image, +1 moved image matches changes)*  
+
+# Mentem
+
+## Creo Mentem
+
+## Intellego Mentem
+
+## Muto Mentem
+
+## Perdo Mentem
+
+## Rego Mentem
+
+# Terram
+
+## Creo Terram
+
+## Intellego Terram
+
+## Muto Terram
+
+## Perdo Terram
+
+## Rego Terram
+
+# Vim
+
+## Creo Vim
+
+## Intellego Vim
+
+## Muto Vim
+
+## Perdo Vim
+
+**Demons Eternal Oblivion (20)**  
+PeVi lvl 20 
+**R:** Voice, **D:** Mom, **T:** Ind  
+Weakens and possibly destroys a demon. If the spell penetrates the demon’s Magic Resistance, the demon loses Might Score equal to the spell lvl = 20.  
+**Mastery:** Penetration x2.  
+*(Base effect, Voice +2)* 
+
+## Rego Vim
+
+**Counterspell**  
+ReVi lvl 10  
+**R:** Voice, **D:** Con, **T:** Ind  
+Suppress a spell cast by another with level less than or equal to half the (level + 5 magnitudes) of the Vim spell = lvl 20.      
+**Mastery:** Fast cast  
+*(Base 3, Voice +2, con +1)*
+
+**Aegis of the Hearth (20)**  
```

Comment:
Those a Core book spells, we shouldn't duplicate the info, except within the character sheet itself if it includes the description of every spell.